design-documents/graph-ql/coverage/Cart.graphqls

```diff
@@ -0,0 +1,58 @@
+type Query {
```

Comment:
please add information about shipping address if we add an item from gift registry;
how can we set gift messages, gift wrapping, printed cart information;
how can we set checkout agreements
`<extension_attributes for="Magento\Quote\Api\Data\PaymentInterface">
        <!-- This is needed to provide type hint to serializer -->
        <attribute code="agreement_ids" type="string[]" />
    </extension_attributes>`